package.json

```diff
@@ -35,6 +35,7 @@
     "@ethersproject/bignumber": "^5.7.0",
     "@google-cloud/secret-manager": "^5.3.0",
     "@google-cloud/tasks": "^5.5.1",
+    "@hashgraph/sdk": "^2.70.0",
```

Comment:
Using this SDK to convert Hedera token address to EVM address, before we save token to database